sheets/coupon_request_api.py

```diff
@@ -605,80 +305,149 @@ def create_coupon_assignment_sheet(self, coupon_req_row):
         )
         return bulk_coupon_sheet
 
-    def create_and_update_sheets(self, processed_requests):
-        """
-        Updates the coupon request sheet and creates a coupon assignment sheet for every successfully
-        processed coupon request.
+    def update_completed_rows(self, success_row_results):
+        for row_result in success_row_results:
+            self.worksheet.update_values(
+                crange="{date_processed_col}{row_index}:{error_col}{row_index}".format(
+                    date_processed_col=self.sheet_metadata.PROCESSED_COL_LETTER,
+                    error_col=self.sheet_metadata.ERROR_COL_LETTER,
+                    row_index=row_result.row_index,
+                ),
+                values=[
+                    [
+                        format_datetime_for_sheet_formula(
+                            row_result.row_db_record.date_completed.astimezone(
+                                settings.SHEETS_DATE_TIMEZONE
+                            )
+                        ),
+                        "",
+                    ]
+                ],
+            )
 
-        Args:
-            processed_requests (list of ProcessedRequest): List of ProcessedRequest objects
+    def post_process_results(self, grouped_row_results):
+        # Create assignment sheets for all newly-processed rows
```

Comment:
Can we make this a docstring?

Reply:
The parent method has an docstring and this comment describes a detail of the specific implementation for this class. I want to leave the parent docstring as-is. Tbh this seems like the kind of thing I should know a solution for but I just don't 😅